### ISSUE TITLE
add self.engine in compiler.py?

### ISSUE THREAD

**Issue by swuecho** [mcfunley/pugsql#5]
make it convenient to get the engine, in case user need it to do some custom query.

**Comment by mcfunley** [comment]
Hey thanks!

Would this be the same as just needing a way to compile a query at runtime? Someone else brought that up. There's not a fantastic way to accomplish this now.

Maybe something like this sketch?

```python
queries = pugsql.compile(path)

# later
queries.add('newquery', sql_string)

results = queries.newquery(**params)
```

`add` is probably too generic of a name for it but that'd be the idea.

**Comment by swuecho** [comment]
This is also helps. but sometimes, for example, I want to run query on each table. the param is the table_name, which sqlalchemy will add extra `''`. the query can not run through pugsql. in situation like this, a connection is necessary.

**Comment by jroxendal** [comment]
I was just looking for something like the `add` above!

**Comment by mcfunley** [comment]
released in 0.2.2

**Comment by potem** [comment]
Could you provide an example in code how this works in the current version?
